### ISSUE TITLE
Put Mock.Random to another package.

### ISSUE THREAD

**Issue by lcjnil** [nuysoft/Mock#195]
Sometimes I just want to use Mock.Random to make some fake data (especially some Chinese features), it would be better if you can put it into another package.

**Comment by gutenye** [comment]
Agreed with @lcjnil , The advantages are easy to contribute, easy to maintain and become a modular code.

See [my question on segmentfault](https://segmentfault.com/q/1010000009815631)

You can use [lerna](https://github.com/lerna/lerna) to create multiple packages in single repository. I can send a PR if you like.

**Comment by gutenye** [comment]
You can use [lerna](https://github.com/lerna/lerna) to create multiple packages in single repository. I can send a PR if you like.
